utils/utils.py

```diff
@@ -5,6 +5,8 @@
 
 from errbot import BotPlugin, botcmd
 
+import config
```

Comment:
@meetmangukiya , is this OK.  I vaguely recall you had concerns about `config` being imported.

Reply:
Yes, please remove this import statement, and use `self.bot_config.COBOT_ROOT` instead